Marlin/src/feature/bedlevel/abl/abl.cpp

```diff
@@ -35,7 +35,7 @@
   #include "../../../lcd/extensible_ui/ui_api.h"
 #endif
 
-xy_int_t bilinear_grid_spacing, bilinear_start;
+xy_float_t bilinear_grid_spacing, bilinear_start;
```

Comment:
I just realized these are also stored in the EEPROM, so if changing these to float is correct, they will also need updated on configuration_store.cpp.

Reply:
Changing these increases the EEPROM usage this needs, but I've pushed new commits to update this.